doc/changes/latest.inc

```diff
@@ -24,6 +24,7 @@ Current (1.2.dev0)
 Enhancements
 ~~~~~~~~~~~~
 - EEGLAB files (saved as MAT versions less than v7.3) can now be imported with :func:`mne.io.read_raw_eeglab` without the optional dependency ``pymatreader`` (:gh:`11006` by `Clemens Brunner`_)
+- Document ``height`` and ``weight`` keys of  ``subject_info`` entry in :class:`mne.Info` (:gh:`10964` by :newcontrib:`Sena Er`)
```

Comment:
```suggestion
- Document ``height`` and ``weight`` keys of  ``subject_info`` entry in :class:`mne.Info` (:gh:`11019` by :newcontrib:`Sena Er`)
```

FYI in the changelog we list the PR number where the problem was fixed, not the issue number where the problem was reported.

Also, this entry isn't really an "enhancement" (which means "new functionality").  The closest fit for missing documentation would be the `Bugs` section. Can you move it, and correct the PR number?

Reply:
Yes, of course. Thanks!